scripts/rootpy-add-include-path

```diff
@@ -0,0 +1,23 @@
+#! /bin/env python
```

Comment:
I wanted to have a small and manageable script. I guess that it might be possible, but it would take me a significant amount of time to avoid screwing up everything.